### PR TITLE
Add _e2e-verify-query function.

### PR DESCRIPTION
**Pull request by ianlewis**

### CODE REVIEW

File: .github/workflows/scripts/e2e-verify.sh [hunk 1]
```diff
@@ -143,7 +143,14 @@ e2e_verify_predicate_builder_id "$ATTESTATION" "https://github.com/slsa-framewor
 e2e_verify_predicate_builderType "$ATTESTATION" "https://github.com/slsa-framework/slsa-github-generator-go@v1"
 
 e2e_verify_predicate_invocation_configSource "$ATTESTATION" "{\"uri\":\"git+https://"github.com/$GITHUB_REPOSITORY"@$GITHUB_REF\",\"digest\":{\"sha1\":\"$GITHUB_SHA\"},\"entryPoint\":\"$GITHUB_WORKFLOW\"}"
-e2e_verify_predicate_invocation_environment "$ATTESTATION" "[\"$GITHUB_ACTOR\",\"$GITHUB_SHA\",\"ubuntu20\",\"X64\",\"$GITHUB_EVENT_NAME\",\"$GITHUB_REF\",\"$GITHUB_REF_TYPE\"]"
+
+e2e_verify_predicate_invocation_environment "$ATTESTATION" "$GITHUB_ACTOR" "github_actor"
```
Comment (laurentsimon) [inline .github/workflows/scripts/e2e-verify.sh:147]: maybe it's more natural to have `"$ATTESTATION" "github_actor" "$GITHUB_ACTOR"`?

**Review by laurentsimon — APPROVED** [review]

File: .github/workflows/scripts/e2e-verify.sh (hunk 1 above)
Comment (ianlewis) [reply .github/workflows/scripts/e2e-verify.sh]: Done.

**Comment by laurentsimon** [comment]
Feel free to merge when you're ready. No further comment on my side

**Comment by ianlewis** [comment]
I think my invitation expired or didn't work or something. I can't merge myself.

**Comment by laurentsimon** [comment]
let me invite you as maintainer again

**Comment by ianlewis** [comment]
Thanks!

**Pull request merged**